packages/@okta/vuepress-theme-prose/const/navbar.const.js

```diff
@@ -448,63 +448,63 @@ export const guides = [
         title: "Authorization",
         subLinks: [
           {
-            title: "Implement authorization by grant type",
+            title: "Implement authz by grant type",
```

Comment:
Authorization is probably clear from context, so how about just:

```suggestion
            title: "Implement by grant type",
```